src/index.html

```diff
@@ -16,7 +16,11 @@
       href="./styles/index.scss"
```

Comment:
The 'href' attribute should link to a CSS file, not an SCSS file. Ensure that the SCSS file is compiled to CSS and linked here.